portal/templates/admin/user-edit.njk

```diff
@@ -205,72 +196,81 @@
-      items: [
-        {
-          value: "true",
-          text: "Notify user of new account?"
-        }
-      ]
-    }) }} #}
-
-      {{ govukInput({
-        attributes: {"data-cy": "firstname"},
-        label: {
-          text: "First name"
-        },
-        classes: "govuk-!-width-one-third",
-        id: "firstname",
-        name: "firstname",
-        value: displayedUser.firstname
-      }) }}
-
-      {{ govukInput({
-        attributes: {"data-cy": "surname"},
-        label: {
-          text: "Surname"
-        },
-        classes: "govuk-!-width-one-third",
-        id: "surname",
-        name: "surname",
-        value: displayedUser.surname
-      }) }}
-
-    {% if _id %}
-      <p class="govuk-body">Bank<br/>{{ displayedUser.bank.name }}</p>
-    {% else %}
-      <div class="govuk-form-group">
-        <label class="govuk-label" for="bank">
-          Bank
-        </label>
-        <span class="govuk-hint">
-          This field is required when the user has a role "Bank staff".
-        </span>
-        <select class="govuk-select govuk-!-width-one-third" id="bank" name="bank" data-cy="bank">
-          <option value="all">Any</option>
-          {% for bank in banks %}
-            <option value="{{ bank.id}}">{{ bank.name }}</option>
-          {% endfor %}
-        </select>
-      </div>
-    {% endif %}
+  {{ govukInput({
+    attributes: {"data-cy": "firstname"},
+    label: {
+      text: "First name"
+    },
+    classes: "govuk-!-width-one-third",
+    id: "firstname",
+    name: "firstname",
+    value: displayedUser.firstname
+  }) }}
 
-    {% if _id %}
+  {{ govukInput({
+    attributes: {"data-cy": "surname"},
+    label: {
+      text: "Surname"
+    },
+    classes: "govuk-!-width-one-third",
+    id: "surname",
+    name: "surname",
+    value: displayedUser.surname
+  }) }}
+  
+  {% set banksList = [
+    {
+          value: "all",
+          text: "Any",
+          selected: true
+    }
+  ] %}
+  {% for bank in banks %}
+   {%
+      set banksList = (banksList.push({
+      value: bank.id,
+      text: bank.name
+    }), banksList)
+   %}
+  {% endfor %}
+  
+  {{ govukSelect({
+    id: "bank",
+    name: "bank",
+    label: {
+      text: "Bank"
+    },
+    hint: {
+      text: "Applicable for Bank Staff users only."
+    },
+    attributes: {
+      "data-cy": "bank"
+    },
+    classes: "govuk-!-width-one-third",
+    items: banksList
+  }) }}
 
-      {{ govukButton({
-        attributes: {"data-cy": "Save"},
-        text: "Save"
-      }) }}
+  {% if _id %}
+    {{ govukButton({
+      attributes: {"data-cy": "Save"},
+      text: "Save"
+    }) }}
 
-    {% else %}
-      {{ govukButton({
-        attributes: {"data-cy": "CreateUser"},
-        text: "Create user"
-      }) }}
-    {% endif %}
+    {{ govukButton({
+        text: "Change password",
+        classes: "govuk-button--secondary",
+        href: "/admin/users/change-password/" + _id,
+        attributes: {
```

Comment:
I don't think having it here is intuitive - it's a secondary button and it can get confused with "Cancel" . The original design had it at the top.